# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/airspyhf/hotfix-bool.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- airspyhf-1.6.8/tools/src/airspyhf_rx.c.vanilla	2026-09-05 00:17:59.008574691 +0300
+++ airspyhf-1.6.8/tools/src/airspyhf_rx.c	2026-09-05 00:18:11.701646845 +0300
@@ -38,9 +38,9 @@
 
 #if !defined __cplusplus
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 #endif
 
--- airspyhf-1.6.8/libairspyhf/src/airspyhf.c.vanilla	2026-09-05 00:18:20.937788505 +0300
+++ airspyhf-1.6.8/libairspyhf/src/airspyhf.c	2026-09-05 00:18:30.648490193 +0300
@@ -34,6 +34,7 @@
 #include <libusb.h>
 #include <pthread.h>
 #include <math.h>
+#include <stdbool.h>
 
 #include "iqbalancer.h"
 #include "airspyhf.h"
@@ -40,9 +40,9 @@
 #include "airspyhf_commands.h"
 
 #ifndef bool
-typedef int bool;
-#define true 1
-#define false 0
+//typedef int bool;
+//#define true 1
+//#define false 0
 #endif
 
 #ifndef M_PI

